class UnauthorizedError(Exception):
    """Исключение отсутствия доступа."""
    pass
